test/suites-ostree/upgrade-ostree.robot

```diff
@@ -0,0 +1,82 @@
+*** Settings ***
```

Comment:
This file is in separate dir so e2e CI job doesn't try to run it against non-ostree system.
When we have e2e for ostree systems we can think if we want to move it back to `suites/` and use tags (or other method to filter tests)